transcript/src/statistics.rs

```diff
@@ -1,37 +1,46 @@
-use crate::{BasicTranscript, Challenge, ForkableTranscript, Transcript};
```

Comment:
This file will be removed in another PR #912 as the way to calculate hashcount is not accurate